packages/fxa-content-server/app/scripts/views/settings/delete_account.js

```diff
@@ -10,21 +10,92 @@ import ServiceMixin from '../mixins/settings-panel-mixin';
 import Session from '../../lib/session';
 import SettingsPanelMixin from '../mixins/service-mixin';
 import Template from 'templates/settings/delete_account.mustache';
+import AttachedClients from '../../models/attached-clients';
+import { CLIENT_TYPE_WEB_SESSION } from '../../lib/constants';
 
 const t = msg => msg;
 
+const LOADING_INDICATOR_BUTTON = '.settings-button.settings-unit-loading';
+const UNIT_DETAILS = '.settings-unit-details';
+
 var View = FormView.extend({
   template: Template,
   className: 'delete-account',
   viewName: 'settings.delete-account',
 
+  initialize(options) {
+    this._attachedClients = options.attachedClients;
+    this._activeSubscriptions = options.activeSubscriptions;
+
+    if (!this._attachedClients) {
+      this._attachedClients = new AttachedClients([], {
+        notifier: options.notifier,
+      });
+    }
+    this._activeSubscriptions = ['test'];
+  },
+
+  _formatTitleAndScope(items) {
+    return items.map(item => {
+      item.title = item.name;
+
+      if (item.scope) item.title += `- ${item.scope}`;
+
+      if (item.clientType === CLIENT_TYPE_WEB_SESSION) {
+        if (item.userAgent) {
+          item.title = this.translate(t('Web Session, %(userAgent)s'), {
+            userAgent: item.userAgent,
+          });
+        } else {
+          item.title = t('Web Session');
+        }
+      }
+
+      return item;
+    });
+  },
+
   setInitialContext(context) {
-    context.set('email', this.getSignedInAccount().get('email'));
+    const clients = this._attachedClients.toJSON();
+    context.set({
+      email: this.getSignedInAccount().get('email'),
+      clients: this._formatTitleAndScope(clients),
+      isPanelOpen: this.isPanelOpen(),
+      subscriptions: this._activeSubscriptions,
+    });
+  },
+
+  openPanel() {
+    this.logViewEvent('open');
+    this.$el.find(UNIT_DETAILS).hide();
+    this.$el.find(LOADING_INDICATOR_BUTTON).show();
+
+    return Promise.all([
+      this._fetchAttachedClients(),
+      this._fetchActiveSubscriptions(),
+    ]).then(() => this.render());
+  },
+
+  _fetchActiveSubscriptions() {
+    const account = this.getSignedInAccount();
+    const start = Date.now();
+    return account.fetchActiveSubscriptions().then(subscriptions => {
+      // do I need to this up anywhere else or does this create `timing.subscriptions`?
+      this.logFlowEvent(`timing.subscriptions.fetch.${Date.now() - start}`);
```

Comment:
To answer your question, nope it will just work how you have it. After it's deployed we can update [the charts](https://sql.telemetry.mozilla.org/queries/49429/source#133062) to pull in the new value.